### ISSUE TITLE
AppFlowy Editor Suddenly Stopped Working in Web platform

### ISSUE THREAD

**Issue by Niralempiric** [AppFlowy-IO/appflowy-editor#44]
Unsupported operation: Platform._operatingSystem

// in this below dart file it point to AppFlowyEditor
The relevant error-causing widget was: 
  AppFlowyEditor AppFlowyEditor:file:///Users/empiric-88/Downloads/mindandme/lib/custom_code/widgets/app_flowy_editor_custom_widget_edit.dart:346:25

**Comment by Niralempiric** [comment]
@LucasXu0 i have also tried new version
this is happening for version 0.1.4 & 0.1.5

**Comment by dmytrohurskyi** [comment]
Same on the Web after updating to 0.1.5 (0.1.4 works OK).
```
The following UnsupportedError was thrown building AppFlowyEditor(dirty, state: _AppFlowyEditorState#7c042):
Unsupported operation: Platform._operatingSystem

When the exception was thrown, this was the stack: 
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 266:49              throw_
dart-sdk/lib/_internal/js_dev_runtime/patch/io_patch.dart 244:5                           _operatingSystem
dart-sdk/lib/io/platform_impl.dart 56:40                                                  get operatingSystem
dart-sdk/lib/io/platform.dart 64:45                                                       get _operatingSystem
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 872:8           get
dart-sdk/lib/io/platform.dart 145:51                                                      get isWindows
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 872:8           get
packages/appflowy_editor/src/service/shortcut_event/shortcut_event.dart 82:25             updateCommand
packages/appflowy_editor/src/service/shortcut_event/shortcut_event.dart 34:5              new
packages/appflowy_editor/src/service/shortcut_event/built_in_shortcut_events.dart 248:3   <fn>
packages/appflowy_editor/src/service/shortcut_event/built_in_shortcut_events.dart 343:16  get builtInShortcutEvents
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 872:8           get
packages/appflowy_editor/src/service/editor_service.dart 167:22                           <fn>
packages/appflowy_editor/src/service/editor_service.dart 123:18                           build
```

```
The following assertion was thrown during a scheduler callback:
Assertion failed: file:///Users/dmytrohurskyi/.pub-cache/hosted/pub.dev/appflowy_editor-0.1.5/lib/src/service/service.dart:13:12
selectionServiceKey.currentState != null &&
        selectionServiceKey.currentState is AppFlowySelectionService
is not true

When the exception was thrown, this was the stack: 
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 266:49      throw_
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 29:3        assertFailed
packages/appflowy_editor/src/service/service.dart 14:29                           get selectionService
packages/appflowy_editor/src/service/editor_service.dart 98:46                    <fn>
```

**Comment by LucasXu0** [comment]
Hey, @dmytrohurskyi @Niralempiric, thanks for reporting. I will release a new version later to fix it. Here's the PR. https://github.com/AppFlowy-IO/appflowy-editor/pull/48

**Comment by LucasXu0** [comment]
@dmytrohurskyi @Niralempiric. FYI, I have released version 0.1.6 to fix this bug.
